content/devdocs/references/p2p_networking.md

```diff
@@ -420,7 +626,16 @@ message header has been omitted and the actual IP addresses have been
 replaced with [RFC5737][] reserved IP addresses.)
 
 {{ <highlight text> }}
-TODO : INTR message hex dump
+IntroductionMessage:
+```
+0e 00 00 00 00 00 00 00 d2 04 00 00 d2 1e 05 00
```

Comment:
No need to add full message . It's too much , especially if messages are long. Annotated message would be enough. Please remove